webapp/templatetags/utils.py

```diff
@@ -25,3 +26,8 @@ def format_date(date):
 @register.filter
 def replace_admin(url):
     return url.replace("admin.insights.ubuntu.com", "blog.ubuntu.com")
+
+
+@register.filter
+def modulo(num, val):
```

Comment:
Why is this needed? Ideally maths would happen in the view rather than the template.